.release-it.json

```diff
@@ -0,0 +1,23 @@
+{
+    "hooks": {
+      "before:init": ["npm test"]
+    },
+    "git": {
+      "requireCleanWorkingDir": true,
+      "commitMessage": "chore: release ${version}",
+      "tagName": "${version}",
```

Comment:
`tagAnnotation` is missing: See the [example repo setup](https://github.com/Amsterdam/ee-docs/blob/develop/.release-it.json#L7)

Reply:
Also please try to order the properties in the same order we have in other projects:
https://github.com/Amsterdam/ee-docs/blob/develop/.release-it.json
https://github.com/Amsterdam/ee-release-it-demo/blob/develop/.release-it.json